### PR TITLE
[docs] Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S *.aip,*.enc -L datas,bu,te`

### CODE REVIEW

**Review by wjs018 — COMMENTED** [review]
All of these typo corrections look accurate. I would approve, but this is making me realize that the python license included in PySceneDetect is pretty old as none of these sections are present in the current python license. 

@Breakthrough would you rather update the license and then merge the rest of the corrections?

File: dist/windows/LICENSE-PYTHON
Comment (Breakthrough) [inline dist/windows/LICENSE-PYTHON:1]: Please revert the changes to this file, this file needs to be reproduced verbatim for this project.

**Comment by Breakthrough** [comment]
Thanks for fixing this up.  @wjs018 I think the license is correct, but I can double check for the next release.  The `dist/windows/LICENSE-PYTHON` should be similar to the LICENSE.txt file on the Python installed on your system.  Is that the same one you were referring to?

**Comment by kianmeng** [comment]
@Breakthrough Done. Please check.

**Review by wjs018 — APPROVED** [review]

**Comment by wjs018** [comment]
@Breakthrough I was comparing to the python license [here](https://docs.python.org/3/license.html). There are probably version differences.

**Review by Breakthrough — APPROVED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:
